build/lint-vars.js

```diff
@@ -27,7 +27,7 @@ function findUnusedVars(dir) {
     process.exit(1)
   }
 
-  console.log(`Finding unused variables in "${dir}"...`)
+  console.log(`\x1b[1m%s\x1b[0m`, `\n➜ Finding unused variables in "${dir}"...`)
```

Comment:
I'd rather not add those unicode stuff because it's not cross-platform.

Also, note that I have made this script a module https://github.com/XhmikosR/find-unused-sass-variables. I just need to publish it after some tweaks and then I'll switch to that here too.

Reply:
So, if you did only the word changes, I'd appreciate it.

Reply:
> I'd rather not add those unicode stuff because it's not cross-platform.

Is there a cross-platform solution you can recommend? I assume you mean Mac and Windows, correct?